music21/expressions.py

```diff
@@ -333,6 +333,10 @@ def __init__(self, content=None):
 
         # this does not do anything if default y is defined
         self.positionPlacement = None
+        # staff number of this TextExpression
+        self.staffKey = staffKey
+        # musicXML <sound> element that is in <direction>, if it exists
+        self.mxSound = mxSound
```

Comment:
These are MusicXML specific notions, so they'll need to be translated into a platform-agnostic format.  What will this do with, say, MIDI or MEI?